### ISSUE TITLE
[Pricing Table]: QA: User is unable to scroll to the bottom content of the page.

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9162]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/ADCMS-2458
- DPO consulting engineer: N/A

-----

### Description

Page Element: Bottom region of the page

Issue: User is unable to scroll to the bottom content of the page.

### Component(s) impacted

Pricing Table

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by proeung** [comment]
@Gopi916 Can you provide steps to reproduce this issue or a video screenshot of the issue you're seeing within the Storybook preview? So far, we're not seeing this scrolling issue within the SB preview or on the AEM side. Moving the severity label down to a lower number for now.

cc: @jkaeser

**Comment by jkaeser** [comment]
> @Gopi916 Can you provide steps to reproduce this issue or a video screenshot of the issue you're seeing within the Storybook preview? So far, we're not seeing this scrolling issue within the SB preview or on the AEM side. Moving the severity label down to a lower number for now.

@proeung You can see this when you navigate the to the [Default Story](https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental/?path=/story/components-pricing-table--default) and attempt to scroll down. The fixed header transition engages, but due to the way browser's automatically scroll when rendered content disappears.

https://user-images.githubusercontent.com/12532727/180861788-b315345c-d3f1-4044-8940-ec4190ccbdbb.mov

**Comment by proeung** [comment]
@jkaeser Okay, this looks like an issue isolated to just the Storybook default preview, is that correct? If so, then we can lower the severity of the issue.

**Comment by gopinathan-n** [comment]
@jkaeser @kennylam Verified in https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental/iframe.html?id=components-pricing-table--default
Still able to replicate the issue.

**Comment by jkaeser** [comment]
> @jkaeser @kennylam Verified in https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental/iframe.html?id=components-pricing-table--default Still able to replicate the issue.

@gopinathan-n You can test the fix for this issue in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9205#issuecomment-1210632691.
